configure

```diff
@@ -76,17 +76,24 @@ set -- "${POSITIONAL[@]}"
 AP_IP=${ARG_AP_IP:-'192.168.10.1'}
 AP_IP_BEGIN=`echo "${AP_IP}" | sed -e 's/\.[0-9]\{1,3\}$//g'`
 MAC_ADDRESS="$(cat /sys/class/net/wlan0/address)"
+COUNTRY="$(wpa_cli -i wlan0 get country)"
 
 # Install dependencies
 sudo apt -y update
 sudo apt -y upgrade
-sudo apt -y install dnsmasq dhcpcd hostapd cron
+sudo apt -y install dnsmasq hostapd
```

Comment:
Why deleting `dhcpcd`?

Reply:
cron: omit as it's no longer needed
dhcpcd: `dhcpcd` and pre-existing [`dpcpcd5`](https://archive.raspberrypi.org/debian/pool/main/d/dhcpcd5/) tend to interfere - thus either we may omit it and go for system default ... or explicitly require dhcpcd5 if you prefer